Apps/Admin/Client/wwwroot/appsettings.json

```diff
@@ -9,5 +9,9 @@
         "Authority": "https://loginproxy.gov.bc.ca/auth/realms/health-gateway-gold",
         "ClientId": "hg-admin-blazor"
     },
+    "TimeZone": {
+        "UnixTimeZoneId": "America/Vancouver",
```

Comment:
We will have to monitor this and see as the code is running on the browser and so we could have other environments that we haven't looked at.  I read at some point that not all timezones were available to the wasm runtime.

Reply:
ok

Reply:
I did some looking into this a little while back and found a discussion on GitHub similar to what you mentioned @sslaws, but they fixed the issue such that there's a decent selection of time zones available now, including the ones we're interested in.